pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -28,14 +29,12 @@ let
   llvmPackages_attrName = "llvmPackages_16";
   stdenv = pkgs.${llvmPackages_attrName}.stdenv;
 
-  upstream-info = (import ./upstream-info.nix).${channel};
-
   # Helper functions for changes that depend on specific versions:
   warnObsoleteVersionConditional = min-version: result:
-    let ungoogled-version = (import ./upstream-info.nix).ungoogled-chromium.version;
+    let min-supported-version = "114";
     in lib.warnIf
-         (lib.versionAtLeast ungoogled-version min-version)
-         "chromium: ungoogled version ${ungoogled-version} is newer than a conditional bounded at ${min-version}. You can safely delete it."
+         (lib.versionAtLeast min-supported-version min-version)
+         "chromium: min-supported-version is newer than a conditional bounded at ${min-version}"
```

Comment:
Of the supported flavors (chromium, chromium-beta, ungoogled-chrome), the ungoogled-chrome channel would always be the most behind for practical reasons (porting the patches takes time).
So once ungoogled-chrome had advanced past a certain version, the expressions could be cleaned up to remove this version.

Now it is most likely the oldest supported electron version, because that is usually 4 versions behind stable (compared to ungoogled-chrome's 1-2).
I will think about implementing this again with some logic that selects the chromium version of the oldest electron version that we support.